src/crimson/os/seastore/lba_mapping.h

```diff
@@ -192,6 +199,8 @@ class LBAMapping {
   // their keys.
   LBACursorRef physical_cursor;
   LBACursorRef indirect_cursor;
+  using LBALeafNode = lba_manager::btree::LBALeafNode;
+  std::optional<child_pos_t<LBALeafNode>> child_pos;
```

Comment:
IIUC:
```c++
The helper structure to be used between the atomic `get_logical_child()` and `link_child()`
in case the logical extent (the child) is absent. It is only related to the direct_cursor.
```